src/main/assembly/assembly.xml

```diff
@@ -0,0 +1,40 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.2.0 https://maven.apache.org/xsd/assembly-2.2.0.xsd">
+    <id>zip</id>
```

Comment:
you missed changing this id as well in `assembly`

Reply:
thanks for pointing it out.